README.md

```diff
@@ -69,9 +78,10 @@ If you want VS Code to automatically close tags while you type, you can install
 
 ## Known `vscode-eslint` issues
 
-1. `Fatal javascript OOM in GC during deserialization`
+1.  `Fatal javascript OOM in GC during deserialization`
```

Comment:
Should these paragraphs maybe also be in the list items?

Reply:
Yes, good call.